demo/fit_a_line/train_ft.py

```diff
@@ -14,7 +14,11 @@ def main():
 
     # network config
     x = paddle.layer.data(name='x', type=paddle.data_type.dense_vector(13))
-    y_predict = paddle.layer.fc(input=x, size=1, act=paddle.activation.Linear())
+    y_predict = paddle.layer.fc(
+        input=x, 
+        size=1, 
+        act=paddle.activation.Linear(), 
+        param_attr=paddle.attr.Param(learning_rate=1e-3))
```

Comment:
`bias_attr` also need to set. Anyway, this is due to the incorrect convertion from old optimizer configuration to new optimizer configuration of paddle.

Reply:
got it, thanks @typhoonzero. will check and update later.

Reply:
> this is due to the incorrect convertion from old optimizer configuration to new optimizer configuration of paddle.

@typhoonzero Can you create a issue with a link indicating the line number which have the problem? This could provide better documentation so that in case in the future we run into it again, we can look at the issue and know what happened. Thanks!

Reply:
@helinwang Sure! I added: https://github.com/PaddlePaddle/Paddle/issues/5005 .